front/app/containers/Admin/pages-menu/SectionToggle/AdminEditButton.tsx

```diff
@@ -7,13 +7,19 @@ import messages from './messages';
 
 interface Props {
   onClick: () => void;
+  testId?: string;
```

Comment:
Same as above comment - get it in another way? `getBytestId` should normally be last resort anyway, although admittedly we are somewhat overusing it currently in other tests, as well.
https://testing-library.com/docs/queries/about/#priority